include/megaapi.h

```diff
@@ -6845,6 +6845,13 @@ class MegaApi
             BUSINESS_STATUS_GRACE_PERIOD = 2
         };
 
+        enum {
+            AFFILIATE_TYPE_INVALID = 0, // legacy mode
+            AFFILIATE_TYPE_ID = 1,
+            AFFILIATE_TYPE_FILE_FOLDER_LINK = 2,
+            AFFILIATE_TYPE_CHAT_LINK = 3
```

Comment:
I suggest to refactor to:
```suggestion
            AFFILIATE_TYPE_FILE_FOLDER = 2,
            AFFILIATE_TYPE_CHAT = 3,
```